<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGitBranchDialog</class>
 <widget class="QDialog" name="QGitBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="operation_label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>New Branch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newBranch_pushButton">
       <property name="text">
        <string>&amp;New Branch</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBranch_pushButton">
       <property name="text">
        <string>&amp;Delete Branches</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Current branch</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="currentBranch_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>New branch:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>New branch:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Working copy parent</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Specified commit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Pick...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Checkout new branch</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>186</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">background: green</string>
           </property>
           <property name="text">
            <string>Create Branch</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Force delete regardless of merge status</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="styleSheet">
            <string notr="true">background: red</string>
           </property>
           <property name="text">
            <string>Delete Branches</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>557</width>
           <height>179</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
